ngraph_bridge/ngraph_builder.cc

```diff
@@ -4129,7 +4129,8 @@ Status Builder::TranslateGraph(
   // Apply additional passes on the nGraph function here.
   //
   ngraph::pass::Manager passes;
-  passes.register_pass<TransposeFolding>();
+  passes.register_pass<pass::TransposeFolding>();
```

Comment:
Removing TransposeFolding would be a separate PR?

Reply:
Yes.